axelrod/tests/unit/test_finite_state_machines.py

```diff
@@ -0,0 +1,236 @@
+"""Tests for Finite State Machine Strategies."""
+import unittest
+
+import axelrod
+from .test_player import TestHeadsUp, TestPlayer
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
```

Comment:
That was my thinking (in addition), I know it's a bit overkill. For the
tests just choose some super simple argument perhaps?

On Tue, 17 May 2016, 05:32 Marc Harper, <notifications@github.com> wrote:

> In axelrod/tests/unit/test_finite_state_machines.py
> <https://github.com/Axelrod-Python/Axelrod/pull/591#discussion_r63463259>:
>
> > @@ -0,0 +1,236 @@
> > +"""Tests for Finite State Machine Strategies."""
> > +import unittest
> > +
> > +import axelrod
> > +from .test_player import TestHeadsUp, TestPlayer
> > +
> > +C, D = axelrod.Actions.C, axelrod.Actions.D
> > +
>
> In addition to the four explicit tests? Right now you can't actually
> instantiate the FSMPlayer without supplying arguments (but we can pick a
> default of course).
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/591/files/0cff3ca068470c7a1cc70531669be0400292c3a9#r63463259>
>